inspector/src/components/sceneExplorer/entities/materialTreeItemComponent.tsx

```diff
@@ -18,6 +19,21 @@ export class MaterialTreeItemComponent extends React.Component<IMaterialTreeItem
     }
 
     render() {
+
+        if(this.props.material.getClassName() === "NodeMaterial") { 
+        
+            return (
+                <div className="materialTools">
+                    <TreeItemLabelComponent label={this.props.material.name} onClick={() => this.props.onClick()} icon={faBrush} color="orange" />
+                    {
+                        <ExtensionsComponent target={this.props.material} extensibilityGroups={this.props.extensibilityGroups} />
+                    }
+                    <div className="icon" onClick={() => {(this.props.material as NodeMaterial).edit()}} title="Node Material Editor" color="white">

```

Comment:
can you change it so we do not need to set the main part twice? (we should just add the icon if it is a nodematerial